src/PrestaShopBundle/Bridge/Smarty/ToolbarFlagsConfigurator.php

```diff
@@ -72,17 +72,17 @@ public function configure(ControllerConfiguration $controllerConfiguration): voi
         $this->initToolbar($controllerConfiguration);
         $this->initPageHeaderToolbar($controllerConfiguration);
 
-        $controllerConfiguration->templatesVars['maintenance_mode'] = !(bool) $this->configuration->get('PS_SHOP_ENABLE');
-        $controllerConfiguration->templatesVars['debug_mode'] = (bool) _PS_MODE_DEV_;
-        $controllerConfiguration->templatesVars['lite_display'] = $controllerConfiguration->liteDisplay;
-        $controllerConfiguration->templatesVars['url_post'] = $this->router->generate('admin_features_index');
-        $controllerConfiguration->templatesVars['show_page_header_toolbar'] = $controllerConfiguration->showPageHeaderToolbar;
-        $controllerConfiguration->templatesVars['page_header_toolbar_title'] = $controllerConfiguration->pageHeaderToolbarTitle;
-        $controllerConfiguration->templatesVars['title'] = $controllerConfiguration->pageHeaderToolbarTitle;
-        $controllerConfiguration->templatesVars['toolbar_btn'] = $controllerConfiguration->pageHeaderToolbarButton;
-        $controllerConfiguration->templatesVars['page_header_toolbar_btn'] = $controllerConfiguration->pageHeaderToolbarButton;
-        $controllerConfiguration->templatesVars['help_link'] = self::HELP_URL . Language::getIsoById($controllerConfiguration->user->getData()->id_lang) . '/doc/'
-            . Tools::getValue('controller') . '?version=' . _PS_VERSION_ . '&country=' . Language::getIsoById($controllerConfiguration->user->getData()->id_lang);
+        $controllerConfiguration->templateVars['maintenance_mode'] = !(bool) $this->configuration->get('PS_SHOP_ENABLE');
+        $controllerConfiguration->templateVars['debug_mode'] = (bool) _PS_MODE_DEV_;
+        $controllerConfiguration->templateVars['lite_display'] = $controllerConfiguration->liteDisplay;
+        $controllerConfiguration->templateVars['url_post'] = $this->router->generate('admin_features_index');
```

Comment:
Hardcoded `Features` route 😝 (not in the scope of this PR)

Reply:
yes. Related to this PR https://github.com/PrestaShop/PrestaShop/pull/28890, but I'll do separate PR for that fix, so i'll add the reminder checkbox in the list here https://github.com/PrestaShop/PrestaShop/issues/28831 and a TODO in code